docs/documentation/updating-the-kit.md

```diff
@@ -13,43 +13,41 @@ Tell us as much as you can about the issue you're having, and the computer and o
+### Updating to version 11
 
-5. In your prototype folder, delete everything except the `app` and `.git` folders.
+1. Make a backup copy of your prototype folder.
 
-6. Copy all the files and folders except the `app` folder from the `govuk-prototype-kit` folder to your prototype.
+2. Make a backup of your prototype folder. You can do this in Finder or Windows Explorer. This may take a few minutes. In the terminal, `cd` to your prototype folder.
 
-7. Replace the `app/config.js` file in your prototype with the `app/config.js` file from the `govuk-prototype-kit` folder.
+3. Run this command:
+ 
+```
+curl -L https://govuk-prototype-kit.herokuapp.com/docs/update.sh | bash
+```
 
-8. Compare your new `config.js` file to the `config.js` file in the backup you made in step 3, and copy over anything you need to from the backup - for example your service name.
 
-9. Copy `app/assets/sass/patterns` from the `govuk-prototype-kit` folder to your prototype.
+4. In a code editor (like Atom) open the file at `update/app/assets/sass/application.scss`.
 
-10. Open the `app/assets/sass/application.scss` file in the `govuk-prototype-kit` folder.
+5. Copy everything until the line that starts with `// Add extra styles here`. 
 
-11. Copy everything down to `// Add extra styles here`, then paste it into the `app/assets/sass/application.scss` file in your prototype so it replaces everything above `// Add extra styles here`.
+6. Open the file at `app/assets/sass/application.scss`. This is the file in your prototype folder, not the one in the update folder. 
 
-12. Save the updated `app/assets/sass/application.scss` file in your prototype.
+7. Delete everything above `// Add extra styles` here and paste what you copied in step 5. Save the file. 
 
-13. Check the [latest Prototype Kit release note](https://github.com/alphagov/govuk-prototype-kit/releases/latest) and follow any guidance on updating your prototype.
+8. In your [terminal](https://govuk-prototype-kit.herokuapp.com/docs/install/requirements.md#terminal), run `npm install`. This may take up to a minute. You can ignore any lines in the log that start with `WARN`.
 
-    If your prototype has not been updated for a long time, you should also follow any guidance in [release notes](https://github.com/alphagov/govuk-prototype-kit/releases) between the version you're updating from and the latest version. You can find out the version you're updating from in the `VERSION.txt` file in your backup folder.
+9. In your terminal, run `npm start`.
 
-14. In your [terminal](/docs/install/requirements.md#terminal), `cd` to your prototype folder.
+10. Check your prototype to see if it works as expected.
 
-15. Run `npm install`.
-
-    This may take up to a minute. You can ignore any lines in the log that start with `WARN`.
-
-16. [Run the kit and check it works](/docs/install/run-the-kit).
+11. Delete the update folder in Finder or Windows Explorer.
```

Comment:
Similar to my earlier comment, it might be clearer to say:
> Delete `update/` in Finder or Windows Explorer.

Reply:
As above.